src/palmatrix-palmabot

```diff
@@ -0,0 +1,56 @@
+---
+title: Palmatrix PalmaBot | SORA Docs
+head:
+  - - meta
+    - name: description
+      content:Explore Palmatrix's PalmaBot to connect Polkaswap to centralized exchanges, enhancing decentralized trading efficiency..
+  - - meta
+    - name: keywords
+      content: SORA builders, SORA, centralised exchange, polkaswap
+---
+
+# Palmatrix PalmaBot
+
+## Introduction
+
+The Palmabot is the first deliverable for Palmatrix as a SORA Builder, per the RPF proposing the infrastructure to implement the PalmaBot. This one-of-a-kind virtual assistant lives on Telegram and enables users to trade on 40 crypto exchanges through a single/unified UI. It has been in production since 2018 and is getting AI features implemented as we speak to enable natural language recognition as soon as possible (we are lucky with the AI boom with ChatGPT and OpenAI).
```

Comment:
```suggestion
The Palmabot is the first deliverable for Palmatrix as a SORA Builder, per the RPF proposing the infrastructure to implement the PalmaBot. This one-of-a-kind virtual assistant lives on Telegram and enables users to trade on 40 crypto exchanges through a single/unified UI. It has been in production since 2018 and is getting AI features implemented as we speak to enable natural language recognition as soon as possible.
```